domains/nishant.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Nishant's website and blog",
+  "repo": "https://github.com/nishch/nishch.github.io",
+  "owner": {
+    "username": "nishch",
+    "email": "nishantchvedi@gmail.com"
+  },
+  "record": {
+    "URL": "https://twistedsoft.com"
```

Comment:
We are currently working on a bug related to url redirections #221. This may take a week to fix. I'd suggest using a cname record instead. If you want to keep it as it is, let me know, I'll merge it in.

Reply:
@phenax thanks for the update, I have updated the PR with `CNAME` record.